deploy/rbac.yaml

```diff
@@ -80,6 +78,20 @@ rules:
   - patch
   - update
   - watch
+- apiGroups:
```

Comment:
Do we need those in cw-rbac.yaml for cluster-wide?

Reply:
https://github.com/percona/percona-server-mysql-operator/pull/725/commits/75bb02dd587209fdce8554203dd9e23ac91360ea